extensions/blocks/social-previews/modal.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Modal styles for Social Previews
+ */
+
+@import '~@automattic/color-studio/dist/color-variables';
+@import '../../../scss/_utilities/mixins/_breakpoint';
+@import '../../../scss/atoms/colors/_colors';
```

Comment:
I haven't seen any other import from this `scss` directory from `extensions`. Can we import it safely? Let's check with somebody from Crew before merging to master but it isn't blocking the merge to our other modal branch.

Reply:
Jetpack Crew answer

> Pretty sure those were only being used by the main Jetpack admin area before. I don't see it happening in other blocks. I guess it wouldn't be a problem as long as wherever the blocks are, so is Jetpack (Atomic). If it's going to wpcom it might be a problem

So, I think we are good to go!